src/components/search/FacetedFilter/FacetedFilter.tsx

```diff
@@ -0,0 +1,75 @@
+import { graphql } from 'gatsby'
+import React, { useState } from 'react'
+import { useSearch } from 'src/sdk/search/useSearch'
+import type { FacetedFilter_FacetsFragment } from '@generated/FacetedFilter_facets.graphql'
+
+import * as styles from './FacetedFilter.module.css'
+
+interface Props {
+  facets: FacetedFilter_FacetsFragment[]
+}
+
+function FacetedFilter({ facets }: Props) {
+  const [index, setIndex] = useState(-1)
+  const { toggleFacet } = useSearch()
+
+  return (
+    <div className={styles.container}>
+      {facets
+        .filter((facet) => facet.type === 'TEXT')
+        .map(({ name, values }, it) => (
+          <div key={`${name}-${it}`}>
+            <button
+              className={styles.button}
+              onClick={() => setIndex(it)}
+              data-testid="facet-filter-header"
```

Comment:
Are these testIds really necessary? Does Cypress provide selectors such as `getByText` or something like that? I think using these selectors enables us to test things based on what the user sees

Reply:
The problem of using `getByText` is that I would pin my tests based on the values itself rather than the structure of the html. This is problematic because this is a starter, and, for each store we change we would have to change the tests

Reply:
Yeees! You're right, I forgot about the very important "starter" part 😅